src/Groups/directproducts.jl

```diff
@@ -141,6 +168,22 @@ end
     as_perm_group(G::DirectProductGroup)
 
 If `G` is direct product of permutations groups, return `G` as permutation group.
+
+# Examples
+```jldoctest
+julia> H = symmetric_group(3)
+Sym( [ 1 .. 3 ] )
+
+julia> K = symmetric_group(2)
+Sym( [ 1 .. 2 ] )
+
+julia> G = direct_product(H,K)
+DirectProduct of
+
+
+julia> as_perm_group(G)
+Group([ (1,2,3), (1,2), (4,5) ])
+```
 """
 function as_perm_group(G::DirectProductGroup)
    if [typeof(H)==PermGroup for H in G.L]==[true for i in 1:length(G.L)]
```

Comment:
Another remark for a future PR you could write: instead of comparing `typeof` it is better to use `isa`, and to check that something holds for all elements of a list, use the `all` function (which short-circuits). 
So something like this (untested) could be done here and in similar cases:
```suggestion
   if all(H -> H isa PermGroup, G.L)
```